<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>743</width>
    <height>515</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MP4视频播放器--测试版</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/new/prefix1/images/0.ico</normaloff>:/new/prefix1/images/0.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QMyselfVideoWidget" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>721</width>
      <height>401</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>420</y>
      <width>721</width>
      <height>58</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pushButton_Open">
         <property name="text">
          <string>打开</string>
         </property>
         <property name="icon">
          <iconset resource="images.qrc">
           <normaloff>:/new/prefix1/images/1.GIF</normaloff>:/new/prefix1/images/1.GIF</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_Play">
         <property name="text">
          <string>播放</string>
         </property>
         <property name="icon">
          <iconset resource="images.qrc">
           <normaloff>:/new/prefix1/images/2.bmp</normaloff>:/new/prefix1/images/2.bmp</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_Pause">
         <property name="text">
          <string>暂停</string>
         </property>
         <property name="icon">
          <iconset resource="images.qrc">
           <normaloff>:/new/prefix1/images/3.bmp</normaloff>:/new/prefix1/images/3.bmp</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_Stop">
         <property name="text">
          <string>停止</string>
         </property>
         <property name="icon">
          <iconset resource="images.qrc">
           <normaloff>:/new/prefix1/images/4.bmp</normaloff>:/new/prefix1/images/4.bmp</iconset>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_6">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_Volumn">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="images.qrc">
           <normaloff>:/new/prefix1/images/volumn.bmp</normaloff>:/new/prefix1/images/volumn.bmp</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="horizontalSlider_Volumn">
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="value">
          <number>40</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_Name">
         <property name="text">
          <string>视频名称</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QSlider" name="horizontalSlider_position">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_Time">
         <property name="text">
          <string>播放进度</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QMyselfVideoWidget</class>
   <extends>QGraphicsView</extends>
   <header location="global">qmyselfvideowidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
